samples/embedded-auth-with-sdk/src/main/resources/templates/select-authenticator.html

```diff
@@ -61,7 +61,7 @@ <h2 class="text-center"><span th:text="${title}"></span></h2>
         <tr th:each="authenticator : ${authenticators}"
             th:if="${authenticator} != 'security_question'"> <!-- not supported by app at this point -->
             <div class="form-check">
-                <input class="form-check-input" type="radio" name="authenticator-type" th:value="${authenticator.type}" checked="true">
```

Comment:
`authenticator.type` is returned `null` for email authenticator. Using `label` should work fine.